data/transport missions.txt

```diff
@@ -1975,3 +1975,42 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			`As you're walking through the spaceport searching for a bar or pub, you notice one person who stands out. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You pause and look at him more closely; he seems oddly familar. Eventually, his gaze settles on you, making eye contact. He blinks a few times in surprise, then begins moving through the crowd towards you.`
+			choice
+				`	(Turn the other way and lose the man in the crowd.)`
+					decline
+				`	(Wait for him to reach you.)`
+				`	(Walk forward to meet him.)`
+			`	When you and the man come face to face, he studies you for a moment. You realize who he is when you hear his voice: "Yes, I thought it was you. It's Captain <last>, correct? I remember you being quite excited to purchase your first ship, even with the outrageously poor interest rate I gave you." It's the broker who lent you the money to purchase your first ship. "No matter," he says. "It seems you've managed."`
+			`	After pausing for a response and not recieving one, he continues: "I've found myself with a problem. One of my... 'clients' hasn't been very proactive in paying their dues on time. This wouldn't be a problem for any normal client, but this man happens to be in charge of a small street gang on <destination>. Of course, there's a simple fix: send a group of paid enforcers to convince the client to be more... accountable, let's say."`
+				`	"And what does this have to do with me?"`
+				`	(Let him continue speaking.)`
```

Comment:
I don't see the need for an inaction choice, especially since the action and inaction both lead to the same result. i.e. just strike this choice.